### PR TITLE
Created menu, sidebar and "logo", needs improvements and optimisation

### PR DESCRIPTION
**Pull request by rcrignon**
Added menu basics, navigation bar and logo zone components with basic graphic chart. 
TODO : Search bar could still be added.

### CODE REVIEW

**Comment by Gui-Yom** [comment]
Please run `npm run lint -- --fix`

**Comment by rcrignon** [comment]
Done, got this message : 
C:\Users\admin\Desktop\INSA\3A\S2\PWEB\front\src\lib\Hamburger.svelte
  5:120  error  Arrow function should not return assignment  no-return-assign
Don't know what the error is exactly

**Comment by Gui-Yom** [comment]
It's ok, just push changes

File: src/App.svelte [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
 
+<Sidebar bind:open/>
+<Navbar bind:sidebar={open}/>
+
 <main>
     <Mosaic {items}/>
 </main>
 
+<svelte:head>
+	<link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet"/>
```
Comment (Gui-Yom) [inline src/App.svelte:22]: This uses an old version of tailwind, replace with :
```svelte
<svelte:head>
	<script src="https://cdn.tailwindcss.com"></script>
</svelte:head>
```

**Review by Gui-Yom — APPROVED** [review]

File: src/App.svelte (hunk 1 above)
Comment (Gui-Yom) [reply src/App.svelte:22]: Nvm you can just remove this since tailwindcss support has been merged

**Pull request merged**